srcpkgs/etlegacy/INSTALL

```diff
@@ -0,0 +1,19 @@
+# INSTALL
+case "${ACTION}" in
+	post)
+		if [ "${UPDATE}" = "no" ]; then
+			echo ""
+			echo "Game depends on original package blobs to run, so lets fetch them"
+			echo ""
+			curl -o /usr/lib/etlegacy/etmain/pak0.pk3 "https://mirror.etlegacy.com/etmain/pak0.pk3" && \
```

Comment:
Doing that install time is bad idea. This should be left as exercise to the player, or buildtime by restricted package.

Reply:
Yea, I knew this would be a wrestle of sort, I'll take a new swing at it :)

Reply:
@Chocimier is it doable to vendor a script for the enduser to manually execute post install time? 
Im hesitant to restrict it as a first resort, since its expected to know xbps-src.
I eyeballed a just nonfree option; would also be considered a misuse and 219MB of bloat.